packages/salesforcedx-vscode-core/src/settings/pushOrDeployOnSave.ts

```diff
@@ -73,15 +73,7 @@ export class DeployQueue {
       const toDeploy = Array.from(this.queue);
       this.queue.clear();
       try {
-        let defaultUsernameorAlias: string | undefined;
-        if (workspaceUtils.hasRootWorkspace()) {
-          defaultUsernameorAlias = await OrgAuthInfo.getDefaultUsernameOrAlias(
-            false
-          );
-        }
-        const orgType = await workspaceContextUtils.getWorkspaceOrgType(
-          defaultUsernameorAlias
-        );
+        const orgType = await workspaceContextUtils.getWorkspaceOrgType();
```

Comment:
On line 19, the import of OrgAuthInfo and workspaceUtils aren't needed and line 19 can be removed.